lib/Frontend/Frontend.cpp

```diff
@@ -509,7 +510,7 @@ bool CompilerInstance::setUpModuleLoaders() {
 
   std::unique_ptr<SerializedModuleLoader> SML =
```

Comment:
So we should avoid creating and adding `SML` here if implicit modules are disabled.